Jenkinsfile.demo

```diff
@@ -68,6 +68,7 @@ node {
 
     // Clean up
     stage 'Clean'
-    sh "docker kill test-container-${env.BUILD_NUMBER}"
-    sh "docker rm test-container-${env.BUILD_NUMBER}"
+    sh "docker kill \$(docker ps -q)"
```

Comment:
This will kill anything running in docker. It will not be possible to run the job concurrently.

Reply:
It doesn't run concurrently (or more than once successfully) in the current version because of linkchecker failing to restart